src/NuGet.Clients/PackageManagement.UI/Resources.resx

```diff
@@ -442,7 +442,8 @@ Packages installed into Website projects will not be restored during build. Cons
     <value>by {0}</value>
   </data>
   <data name="Text_Downloads" xml:space="preserve">
-    <value>{0} downloads</value>
+    <value> downloads</value>
```

Comment:
lets do the {0} downloads as the resource and use the parsing code that find the {0} and removes it, and then builds the run around it. At a minimum move the space into the control